packages/plugins/i18n/server/src/controllers/validate-locale-creation.ts

```diff
@@ -23,7 +23,7 @@ const validateLocaleCreation: Common.MiddlewareHandler = async (ctx, next) => {
     return next();
   }
 
-  const locale = get('plugins.i18n.locale', query);
+  const locale = get('locale', query) ?? get('locale', body);
```

Comment:
```suggestion
  const locale = get('locale', query) || get('locale', body);
```
Not actually a suggestion, I think I prefer it as you've written it, but just want to be totally sure we're clear about the behaviour we want. For example:

```
Method: POST,
url:http://localhost/create?locale=
body:{
    "locale":"en"
}
``` 

With ?? it may use locale of empty string, with || it will use 'en'

Reply:
(haven't actually tested it, it could be worrying about nothing depending on how the params are converted and received at this point)

Reply:
makes sense!

Reply:
So do we want to ignore empty strings if we receive it? In that case, we may want to add a third case, so:

`const locale = get('locale', query) || get('locale', body) || undefined` to ensure we don't end up with locale === ''